src/Simplic.OxS.ModelDefinition/ModelDefinition.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Simplic.OxS.ModelDefinition
+{
+    public class ModelDefinition
+    {
+        public string? Title { get; set; }
+
+        public string? Model { get; set; }
+
+        public string SourceUrl { get; set; }
+
+        public Operations Operations { get; set; }
+            = new Operations();
+
+        public IList<DataSource> DataSources { get; set; }
+
+        public IList<PropertyDefinition> Properties { get; set; }
```

Comment:
Why don't you initialize this lists, but the one below? If they should not be initialized, use `?`